app/frontend/src/components/settings/lists/StackedListIcon.js

```diff
@@ -13,13 +13,16 @@ function StackedListIcon({thumbnails, favorite}){
     }, [thumbnails, favorite]);
 
     const useImageOrPlaceholder = (image, index) => {
-        if(image && image.base64data && image.base64data !== "" && index !== 0){

```

Comment:
If I remember correctly it is possible to use if(image?.base64data !== "" && ... )
with the same outcome.
Nothing that needs changing but sometimes useful to know especially inside of react components where a variable may not be loaded yet.